indico/modules/categories/templates/management/category_protection.html

```diff
@@ -11,7 +11,7 @@
     {{ form_row(form.permissions, skip_label=true) }}
     {{ form_rows(form, skip=(form._event_creation_fields + ('permissions',)),
                  widget_attrs={'own_no_access_contact': {'placeholder': category.no_access_contact}}) }}
-    {% call form_fieldset(_('Event creation settings')) %}
+    {% call form_fieldset(_('Event settings')) %}
```

Comment:
Aren't those still creation-related settings? Sure, technically moving an event isn't creating it, but in the end it's "created" in the category that way..